webclient.MediaContextTest intermittent failure

```
Caused by: java.util.concurrent.ExecutionException: io.netty.handler.timeout.ReadTimeoutException
	at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:395)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1999)
	at io.helidon.tests.integration.webclient.MediaContextTest.lambda$testInputStreamSameThreadTestContentAs$21(MediaContextTest.java:208)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:55)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:37)
	at org.junit.jupiter.api.Assertions.assertThrows(Assertions.java:2952)
	at io.helidon.tests.integration.webclient.MediaContextTest.testInputStreamSameThreadTestContentAs(MediaContextTest.java:195)
	... 62 more
Caused by: io.netty.handler.timeout.ReadTimeoutException
```

--- a/webclient/webclient/src/main/java/io/helidon/webclient/NettyClientHandler.java
+++ b/webclient/webclient/src/main/java/io/helidon/webclient/NettyClientHandler.java
@@ -290,7 +290,6 @@ class NettyClientHandler extends SimpleChannelInboundHandler<HttpObject> {
                 WebClientServiceResponse clientServiceResponse = channel.attr(SERVICE_RESPONSE).get();
                 CompletableFuture<WebClientServiceResponse> requestComplete = channel.attr(COMPLETED).get();
                 requestComplete.complete(clientServiceResponse);
-                channel.config().setAutoRead(true);
                 WebClientResponse response = channel.attr(RESPONSE).get();
                 String connection = response.headers().first(Http.Header.CONNECTION)
                         .orElseGet(HttpHeaderValues.CLOSE::toString);
@@ -298,7 +297,7 @@ class NettyClientHandler extends SimpleChannelInboundHandler<HttpObject> {
                     ctx.close()
                             .addListener(future -> {
                                 if (future.isSuccess()) {
-                                    LOGGER.finest(() -> "Response from has been closed.");
+                                    LOGGER.finest(() -> "Response from the server has been closed.");
                                     cf.complete(null);
                                 } else {
                                     LOGGER.log(Level.SEVERE,
@@ -310,6 +309,7 @@ class NettyClientHandler extends SimpleChannelInboundHandler<HttpObject> {
                 } else {
                     channel.attr(IN_USE).get().set(false);
                     cf.complete(null);
+                    channel.read();
                 }
                 publisher.complete();
             }

